src/test/msgr/CMakeLists.txt

```diff
@@ -16,15 +16,15 @@ add_executable(ceph_test_async_networkstack
   test_async_networkstack.cc
   $<TARGET_OBJECTS:unit-main>
   )
-target_link_libraries(ceph_test_async_networkstack global ${CRYPTO_LIBS} ${BLKID_LIBRARIES} ${CMAKE_DL_LIBS} ${UNITTEST_LIBS})
+target_link_libraries(ceph_test_async_networkstack global-static ${CRYPTO_LIBS} ${BLKID_LIBRARIES} ${CMAKE_DL_LIBS} ${UNITTEST_LIBS})
 
 #ceph_perf_msgr_server
 add_executable(ceph_perf_msgr_server perf_msgr_server.cc)
-target_link_libraries(ceph_perf_msgr_server os global ${UNITTEST_LIBS})
+target_link_libraries(ceph_perf_msgr_server os global-static ${UNITTEST_LIBS})
```

Comment:
can we expose `dpdk::dpdk` as an `INTERFACE_LINK_LIBRARIES` exposed by os and ceph-common ?

Reply:
ceph-common uses common-msg-objs. When WITH_DPDK is defined, common-msg-objs links common_async_dpdk. Extracting the dpdk separately will cause the common_async_dpdk link to fail.